contrib/travis.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh -ex
+
+if [ -n "$HOST" ]
+then
+    export USE_HOST="--host=$HOST"
+fi
+if [ "$HOST" = "i686-linux-gnu" ]
+then
+    export CC="$CC -m32"
+fi
+if [ "$TRAVIS_OS_NAME" = "osx" ] && [ "$TRAVIS_COMPILER" = "gcc" ]
+then
+    export CC="gcc-9"
+fi
+
+./configure \
+    --enable-experimental="$EXPERIMENTAL" --enable-endomorphism="$ENDOMORPHISM" \
+    --with-field="$FIELD" --with-bignum="$BIGNUM" --with-asm="$ASM" --with-scalar="$SCALAR" \
+    --enable-ecmult-static-precomputation="$STATICPRECOMPUTATION" --with-ecmult-gen-precision="$ECMULTGENPRECISION" \
+    --enable-module-ecdh="$ECDH" --enable-module-recovery="$RECOVERY" "$EXTRAFLAGS" "$USE_HOST"
+
+if [ -n "$BUILD" ]
+then
+    make -j2 "$BUILD"
+fi
+if [ -n "$VALGRIND" ]
+then
+   make -j2
+# the `--error-exitcode` is required to make the test fail if valgrind found errors, otherwise it'll return 0 (http://valgrind.org/docs/manual/manual-core.html)
+   valgrind --error-exitcode=42 ./tests 16
+   valgrind --error-exitcode=42 ./exhaustive_tests
+   fi
+if [ -n "$BENCH" ]
+then
+    if [ -n "$VALGRIND" ]
+    then
+        EXEC='./libtool --mode=execute valgrind --error-exitcode=42'
+    else
+        EXEC=
+    fi
```

Comment:
I first wanted to write it would be cleaner so set `SECP256K1_BENCH_ITERS=2` here, for clarity. But I'm not entirely sure. If we set it here, it won't be configurable.  Bu then we should at least add a comment that the caller should set the ITERS to a low value. What do you think?

Reply:
Hmm I think this script is 100% travis oriented anyway. so I'm not sure there's a meaningful difference.
If the script was written in a way that we could also run it locally pretty easily then maybe it would have (although the amount of env vars make it pretty hard to run locally)

Reply:
> Hmm I think this script is 100% travis oriented anyway. so I'm not sure there's a meaningful difference.

Sure, and no need to change that right now. (Even though we want to change it in the future.) My main concern is that you don't see the low iteration count when looking at the script. Env variables are great in hiding details. It looks like we're just invoking the full benchmarks.  My maybe a comment is indeed the best.